utils/form.ts

```
@@ -0,0 +1,22 @@
+import type { DestForm, SenderForm } from '~/scripts/forms/schema';
+
+export const convertFormToTemplateInput = (form: DestForm & SenderForm) => {
+  const data: Omit<DestForm & SenderForm, 'destZipcode' | 'destHonorific'> & {
+    destZipcode?: string;
+    destHonorific?: string;
+  } = { ...form };
+
+  const destZipcode = data.destZipcode?.replace('-', '') ?? '';
+  data.destName += ` ${data.destHonorific ?? ''}`;
+  delete data.destZipcode;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)